src/commands/dataconnect-sdk-generate.ts

```diff
@@ -31,7 +32,10 @@ export const command = new Command("dataconnect:sdk:generate")
       if (!hasGeneratables) {
         logger.warn("No generated SDKs have been declared in connector.yaml files.");
         logger.warn(
-          "See https://firebase.google.com/docs/data-connect/quickstart#configure-sdk-outputs for examples of how to configure generated SDKs.",
+          `Run ${clc.bold("firebase init dataconnect:sdk")} to configure a generated SDK.`,
+        );
+        logger.warn(
+          `See https://firebase.google.com/docs/data-connect/quickstart#configure-sdk-outputs for more details of how to configure generated SDKs.`,
```

Comment:
@markarndt FYI on the link.

It doesn't work right now. If you have a "future" link, that would be neat.

Reply:
Dang, this link previously worked, looks like we edited that page recently. Linking to https://firebase.google.com/docs/data-connect/gp/web-sdk instead for now, but feel free to edit suggest a better link @markarndt